src/dex-ui/pages/Pools/Pools.tsx

```diff
@@ -141,60 +200,113 @@ const Pools = (): JSX.Element => {
       : [];
   };
 
+  const getHashScanLink = useCallback(() => {
```

Comment:
We should move this into a shared utils file and import the function into the Pools (this) and Swap component. The logic is the same for both components.